kong.conf.default

```diff
@@ -269,6 +269,13 @@
                                  # Similarly to `error_template_html`, the template
                                  # is required to contain one single `%s` placeholder for
                                  # the error message.
+
+#worker_events_max_payload =     # Specifies the max payload size the `worker_events` can
+                                 # accept.
+                                 # Default is 65535. The max allowed is 16777215.
+                                 # **Note:** increasing this value has potential
+                                 # negative impact on Kong's response latency and memory usage.
```

Comment:
```suggestion
#worker_events_max_payload = 65535  # Specifies the max payload size the 
                                    #`worker_events` can accept. The max allowed is 
                                    # 16777215.
                                    # **Note:** Increasing this value has potential
                                    # negative impact on Kong's response latency and memory usage.
```
Default value doesn't need to be in the description, it should be in the first line.

Reply:
done. thanks.